source/documentation/getting_started/get_started.erb

```diff
@@ -135,6 +139,89 @@ You can practice deploying an app by deploying a test static HTML page.
 
     If you do not specify the app name in `cf push`, the name specified in the manifest file is used.
 
-    The static HTML page is now available at your [app domain](/orgs_spaces_users.html#regions).
+The static HTML page is now available at your [app domain](/orgs_spaces_users.html#regions).
+
+For a production app, you should read the [production checklist](deploying_apps.html#production-checklist).
+
+## Deploy a test static HTML page using Cloud Foundry API v3
+
+You can practice deploying an app by deploying a test static HTML page.
+
+The following instructions tell you how to deploy a static HTML page using [v3 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v3). This version of the API allows you to push an app with no downtime without needing to use a [blue-green deployment](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html).
+
+If you are using [v2 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v2), refer to the documentation on [deploying a test static HTML page using Cloud Foundry API v2](get_started.html#deploy-a-test-static-html-page-using-cloud-foundry-api-v2).
+
+<br>
+
+<%= warning_text('Cloud Foundry API v3 is still in beta.') %>
```

Comment:
Warning about this is good, and I feel we would benefit from guiding the user on whether to use V2 vs V3. Although I appreciate it's hard to write that without having to revisit this as V3 becomes finalised.

Reply:
@46bit at the moment we have a link at the end of this section to https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html. This contains https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html#limitations. Should we put this link into this section? You cannot put links into warnings (I don't think?) so it would have to be separate.